lib/sitemap_generator/railtie.rb

```diff
@@ -1,7 +1,7 @@
 module SitemapGenerator
   class Railtie < Rails::Railtie
     rake_tasks do
-      require File.expand_path('../tasks', __FILE__)
+      load "tasks/sitemap_generator_tasks.rake"
```

Comment:
Yes this looks more correct.  I don't know how it would have done anything before because ../tasks/railtie.rb isn't a valid path??

However, I'd prefer to keep the `File.expand_path` call so we know the path is correct.